packages/xml-toolkit/lib/extension.js

```diff
@@ -16,21 +17,37 @@ function activate() {
     },
   };
 
+  const meta = require(resolve(__dirname, "..", "package.json"));
+  const outChannel = window.createOutputChannel(meta.displayName);
   let clientOptions = {
     documentSelector: [{ scheme: "file", language: "xml" }],
     synchronize: {
       fileEvents: workspace.createFileSystemWatcher("**/*.xml"),
     },
+    outputChannel: outChannel,
   };
 
   client = new LanguageClient(
-    "XMLforVSCode",
-    "XML For VSCode",
+    meta.name,
+    meta.displayName,
     serverOptions,
     clientOptions
   );
 
+  // https://github.com/microsoft/vscode-languageserver-node/blob/bb85e80617c7f0cb64dad9dad852219e2da7c4c9/jsonrpc/src/node/test/connection.test.ts#L484-L507
```

Comment:
What does this link point to? Is there documentation for the feature instead of linking to a test?
And should this comment be on this line or the next one?

Reply:
Thanks its left over I used for reference I will remove